build/depends.py

```diff
@@ -489,7 +489,7 @@ def sources(self, build):
 
     def configure(self, build, conf):
         build.env.Append(CPPDEFINES=['KS_STR_ENCODING_NONE'])
-        build.env.Append(CPPPATH="#lib/kaitai")        
+        build.env.Append(CPPPATH="#lib/kaitai")
```

Comment:
Can you remove these unrelated changes (possibly put them into a separate PR) so it gets easier to review?

Reply:
#2367